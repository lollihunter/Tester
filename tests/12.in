20 20 20
1 1
1 18
2 4
13 1
3 14
2 11
20 18
7 10
5 17
15 10
11 16
5 16
18 11
18 1
19 15
17 17
18 7
9 9
9 11
20 20
