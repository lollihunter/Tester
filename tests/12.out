9
1 3 8 19 10 13 15 7 20 
